### ISSUE TITLE
Results from Pretrained RRC weights seems to be errorneous.

### ISSUE THREAD

**Issue by joonjeon** [xiaohaoChen/rrc_detection#27]
Hi?

I have just run the RRC with pretrained .caffemodel weights from Google Drive,
but it seems there are too many bounding boxes.

I am pretty sure that KITTI **does not care about confidence scores of bounding boxes** when performing evaluation on 2D object detection,
so in order to have competitive mAP on KITTI car dataset, one can intuitively think that the number of detected bounding boxes, regardless of scores, must be low for every test set image.

However, when running `wc -l` command on detection results in .txt format, the results are as follows:
```Shell
      34 000000.txt
       3 000001.txt
       8 000002.txt
      26 000003.txt
      44 000004.txt
       6 000005.txt
       3 000006.txt
       4 000007.txt
       9 000008.txt
      70 000009.txt
       4 000010.txt
      40 000011.txt
...
```

For some images, **including those that are not enlisted in shell snippet above**, more than 20 bounding boxes are detected, although there are less than 10 cars in each of those images.
This could result in extremely high false positives, thereby a low mAP...

Could you give me any explanation for this as soon as possible?
Thanks in advance.

**Comment by joonjeon** [comment]
The LMDBs might be set incorrectly.

Training set is supposed to cover all 7481 images, but in the current git repo, it is specified to be trained upon only 503 images. Perhaps this is the cause.

Hope this helps.

**Comment by xiaohaoChen** [comment]
As far as I'm concerned, KITTI **cares about confidence scores** of bounding boxes when performing evaluation on 2D object detection.
For the second question, many of the boxes with low mAP with ground truth have low confidece scores.
